packages/devtools_app/lib/src/shared/analytics/constants.dart

```diff
@@ -94,6 +95,9 @@ const importFileDiffSecond = 'importFileDiffSecond';
 const analyzeSingle = 'analyzeSingle';
 const analyzeDiff = 'analyzeDiff';
 
+// Deep Link Tools UX actions:
+const selectProject = 'selectProject';
```

Comment:
consider adding a part file under `analytics/constants/` for deep link analytics events